test/run-tests.py

```diff
@@ -265,11 +327,16 @@ def CreateRoundtripInfo(self, fold_exprs):
     result.input_ = self.input_
     result.expected_stdout = ''
     result.expected_stderr = ''
+
     # TODO(binji): It's kind of cheesy to keep the enable flag based on prefix.
     # Maybe it would be nicer to add a new directive ENABLE instead.
-    result.flags = [flag for flag in self.flags if flag.startswith('--enable')]
+    old_cmd = self.cmds[0]
+    new_cmd = result.cmds[0]
+    new_cmd.flags = [flag for flag in old_cmd.flags
```

Comment:
i think its ok to use single char variable names in list comprehensions, especially if it means not having to wrap them.

Reply:
Done.